analyses/molecular-subtyping-integrate/01-integrate-subtyping.Rmd

```diff
@@ -0,0 +1,210 @@
+---
+title: "Integrate molecular subtyping results"
+output: 
+  html_notebook:
+    toc: true
+    toc_float: true
+author: Krutika Gaonkar for D3b
+date: 2020
+---
+
+The purpose of this notebook is to integrate molecular subtyping results from 
+[molecular-subtyping-pathology](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-pathology) with `pbta-histologies-base.tsv`.
+
+Here we will use `pbta-histologies-base.tsv` in which integrated_diagnosis,Notes and molecular_subtype are all NA. Through all the following molecular subtyping modules:
+
+- molecular-subtyping-MB
+- molecular-subtyping-CRANIO
+- molecular-subtyping-EPN
+- molecular-subtyping-embryonal
+- molecular-subtyping-EWS
+- molecular-subtyping-neurocytoma
+- molecular-subtyping-HGG
+- molecular-subtyping-LGAT
+- molecular-subtyping-pathology
+
+we gathere and update molecular-subtype AND integrated_diagnosis AND broad_histology AND short_histology for these histologies.
+
+In this notebook we will add the molecular subtyping done as part as OpenPBTA to the base histology file to create the `pbta-histologies.tsv` for the same release. 
```

Comment:
```suggestion
In this notebook, we will add the molecular subtyping updated in the `pbta-histologies-base.tsv` file to update the `pbta-histologies.tsv` file. 
```

Reply:
> subtyping updated in the `pbta-histologies-base.tsv` 

This is not entirely true I mean `pbta-histologies-base.tsv` file is used but is not updated through the subtyping modules. The subtyping file is saved in 
compiled_subtyping<-read_tsv(file.path("..", "molecular-subtyping-pathology", "results", "compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv")) as part of the molecular-subtyping-pathology module.

We directly use compiled_molecular_subtypes_with_clinical_pathology_feedback.tsv to update `pbta-histologies.tsv` file

Reply:
Ah. Okay. Can you describe that here then?

Reply:
oh I forgot to commit the description 😅

Let me know if it makes sense, should I add in my workflow diagram if it's useful in the README or code or this looks ok ?

Reply:
Sure! Why not add the diagram! Would probably be good to have that in this description.

Reply:
done!